ironfish/src/network/peerNetwork.test.ts

```diff
@@ -173,6 +173,27 @@ describe('PeerNetwork', () => {
   describe('handles requests for compact blocks', () => {
     const nodeTest = createNodeTest()
 
+    function expectGetCompactBlockResponseToMatch(
```

Comment:
Do you want to export these from their respective test files? I think they're duplicated, right?

Reply:
Exporting from test files seems like a bad practice, so I've moved these functions to network test utilities